src/index.ts

```diff
@@ -102,6 +102,13 @@ export default class OtaClient {
         return this.locale;
     }
 
+    /**
+     * List manifest timestamp of distribution
+     */
+    async getManifestTimestamp(): Promise<number> {
```

Comment:
Could you please extend the current set of tests and add one more for this method?

Reply:
Added test